app/views/partials/view-claim/claim.html

```diff
@@ -54,7 +54,9 @@
           <select id="claim-expense-{{ expense.ClaimExpenseId }}-status" data-id="{{ expense.ClaimExpenseId }}" name="claim-expense-{{ expense.ClaimExpenseId }}-status"
             class="form-control action-select claim-expense-status">
             <option {% if expense['Status'] == undefined %} selected {% endif %}>Select</option>
+            {% if expense['ExpenseType'] != 'car' and expense['ExpenseType'] != 'train' %}
```

Comment:
To be tested:
[ ] Car expense on advance claim - Approve shouldn't appear
[ ] Train expense on advance claim - Approve shouldn't appear
[ ] Car expense on past claim - Approve shouldn't appear
[ ] Train expense on past claim - Approve should appear